examples/advanced/main.tf

```diff
@@ -107,16 +107,45 @@ module "buckets" {
   ]
 }
 
+##############################################################################
+# Create CBR Zone
```

Comment:
Does this make sense? Why would you create a CBR zone for the cloud logs instance and create a rule that allows it access itself?

Reply:
Generally we create a VPC zone to say only traffic from within a VPC can access the cloud logs instance. You already have schematics as a zone, so thats probably enough to show in the example how CBRs can be used